.github/workflows/phpunit.yml

```diff
@@ -0,0 +1,102 @@
+name: "PHPUnit tests"
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  phpunit:
+    name: "PHPUnit tests"
+
+    runs-on: ${{ matrix.operating-system }}
+
+    strategy:
+      matrix:
+        dependencies:
+          - "lowest"
+          - "highest"
+          - "locked"
+        php-version:
+          - "7.3"
```

Comment:
Note: PHP 7.2 is gone-gone-gone, so I removed it as per https://www.php.net/supported-versions.php

Also, no lib that is compatible with PHP 7.2 also works with PDO changes of PHP 8.x.